tests/fixtures/qasm2/shared15.qasm

```diff
@@ -2,6 +2,7 @@ OPENQASM 2.0;
 include "qelib1.inc";
 gate iswap q0,q1 { s q0; s q1; h q0; cx q0,q1; cx q1,q0; h q1; }
 qreg q[4];
+creg c[2];
```

Comment:
same thing, why is including `creg` necessary if `creg` is never referenced again?

Reply:
same reason, added measurement here